test/unit/CodeGenerator/Filesystem/FilePutContentsFileWriterTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\CodeGenerator\Filesystem;
```

Comment:
The test needs to be in the OnMoon\OpenApiServerBundle\Test\Functional namespace